docs/getting_started/cli.md

```diff
@@ -0,0 +1,31 @@
+# Command Line Interface (CLI) Parameters
+
+```
+Usage: amber [OPTIONS] [INPUT] [OUTPUT]
+
+Arguments:
+  [INPUT]   '-' to read from stdin
+  [OUTPUT]  '-' to output to stdout, '--silent' to discard
+
+Options:
+  -e, --eval <EVAL>     Code to evaluate
+      --docs            Generate docs (OUTPUT is dir instead, default: `docs/`)
+      --disable-format  Don't format the output file
+      --minify          Minify the resulting code
+  -h, --help            Print help
+  -V, --version         Print version
+```
```

Comment:
This help message will change pretty frequently. I think that we shouldn't include it here

Reply:
I was including as reference so people can understand what are the parameters, the man pages usually have this kind of output.
We can update after a release if there are changes.

Reply:
Can we write somewhere that this is just a reference? Also what do you think about it @b1ek @KrosFire @mks-h

Reply:
we can put on top "as per version 0.3.5 this is the help output"

Reply:
I don't know if people will use it and if we will always remember to update this. This is auto generated and I think that standard human behaviour is to use `-h` flag when one doesn't know how to use the cli

Reply:
i think we should just tell people to install the cli and use `--help`. this will be misleading especially if the user has a different version installed

Reply:
the documentation is for the actual version and not the development release.
at every release the plan is to update the documentation so that section will be updated too, right now I added the version where it was generated.
think that any documentation as this output shown as reference, in this way you don't have to install the tool.